cfgov/unprocessed/apps/tccp/css/main.less

```diff
@@ -38,6 +38,37 @@
   }
 }
 
+.a-checkbox + .a-label > .a-label_helper {
+  display: none;
+}
+
+.a-checkbox:checked + .a-label > .a-label_helper {
+  display: grid;
+  grid-column-start: 1;
+  grid-column-end: 3;
+  margin-top: 1em;
+  margin-bottom: 1em;
+}
+
+.a-label_helper .m-list {
+  list-style: square;
```

Comment:
@caheberer the designs show circle bullet points but I made them squares to match the [DS guidance](https://cfpb.github.io/design-system/components/lists#bulleted-list).